libs/langchain-azure-openai/src/legacy.ts

```diff
@@ -0,0 +1,355 @@
+import {
```

Comment:
Let's leave this out! Unless you have a reason to keep it?

Reply:
@jacoblee93  I would like this to continue for now, because I want to keep it consistent with @langchain/openai package. When we retire the legacy class in that package, we could remove it from here also. Would that be ok? But, if you really want to drop this out, I can remove it. Please suggest.

Reply:
Let's cut it - we want to steer people away from it. It's already been dropped from the docs and marked as deprecated in the other package.